mongodb-plugin/src/main/java/io/cdap/plugin/MongoDBConfig.java

```diff
@@ -18,9 +18,12 @@
 
 import com.google.common.base.Strings;
```

Comment:
please add unit tests for both configs

Reply:
Added:
1. [MongoDBConfigTest](https://github.com/vladhlinskiy/database-plugins/pull/14/files#diff-0581c35cf54168061f5e19b52987a299R29)
2. [MongoDBSourceConfigTest](https://github.com/vladhlinskiy/database-plugins/pull/14/files#diff-358da390ff068ee56dc555a9c56e722bR29)